src/bookshelf/extras.ts

```diff
@@ -11,6 +11,7 @@ export interface Model extends BModel<any> {
   id: any; // TODO IMPROVE REAL TYPE
   attributes: any;
   relations:  RelationsObject;
+  models: any;
```

Comment:
This shouldn't be here: a Model from Bookshelf doesn't have an internal models property (remember these extra interfaces exposes private properties because we use them to analyze the object)